inst/BS5/assets/pkgdown.scss

```diff
@@ -20,6 +20,12 @@
 
 /* navbar =================================================================== */
 
+.navbar {
+  background: mix(mix($body-bg, $body-color, 95%), $primary, 98%);
```

Comment:
Probably should make these variables so user can override? (Like footer colour below)